homeassistant/components/alexa/handlers.py

```diff
@@ -475,13 +475,6 @@ async def async_api_unlock(
     context: ha.Context,
 ) -> AlexaResponse:
     """Process an unlock request."""
```

Comment:
```suggestion
    """Process an unlock request."""
    if config.locale not in {
        "ar-SA",
        "de-DE",
        "en-AU",
        "en-CA",
        "en-GB",
        "en-IN",
        "en-US",
        "es-ES",
        "es-MX",
        "es-US",
        "fr-CA",
        "fr-FR",
        "hi-IN",
        "it-IT",
        "ja-JP",
        "pt-BR",
    }:
        msg = (
            "The unlock directive is not supported for the following locales:"
            f" {config.locale}"
        )
        raise AlexaInvalidDirectiveError(msg)

```

Reply:
Why did you mark this as resolved?

Reply:
Because it is synchronized with amazon alexa spec, it is better than my proposal which remove this filter.
But in my opinion this filter is also needed by the lock directive.

Reply:
Okay, where is synchronised. Van we add a test?